rcljava/src/main/java/org/ros2/rcljava/time/TimeSource.java

```diff
@@ -33,6 +33,15 @@
 
 import java.util.concurrent.LinkedBlockingQueue;
 
+/**
+ * Implementation of the ROS time source abstraction.
+ *
+ * The abstraction uses a subscription to a "/clock" topic as described in
+ * the design document
+ * <a href="http://design.ros2.org/articles/clock_and_time.html">Clock and Time</a>.
```

Comment:
Mentioning `use_sim_time` in the docblock may be a good idea.

Reply:
Can you take another look at this dockblock? I've refactored it, adding more detail bf546a2

Reply:
SGTM